offerzen/global/employers/employers@4.0.js

```diff
@@ -9,44 +9,44 @@ window.$loaded(function (window, document, $, undefined) {
         $parsleyLoaded(cb)
       }, 50)
     }
-  })();
-
-  window.$parsleyLoaded(function (window, document, parsley) {
```

Comment:
Why is this being removed? It's easier and less error-prone to wrap everything in parsleyLoaded than multiple parts.